front-end/src/Pages/JoinRoom.tsx

```diff
@@ -33,6 +33,7 @@ export default function JoinRoom(props: IJoinRoomProps) {
         } else {
           try {
             addToRoom(roomId)
+            history.push(`/redirecting`)
```

Comment:
What's this for?

Reply:
Yeah, directly pushing the room to the history didn't seem to fully re-render the page causing the older user list to be displayed, so this redirect solved it. Maybe we can add a loading screen there.